pupil_src/shared_modules/head_pose_tracker/controller/online_controller.py

```diff
@@ -47,6 +47,8 @@ def _on_recent_events(self, events):
             self._calculate_current_markers(events["frame"])
             self._calculate_current_pose(events["frame"])
             self._save_key_markers()
+            events["head_pose"] = self._create_position_events(
```

Comment:
Since you renamed the events-key from `head_position` to `head_pose` I suggest you also rename the function from `_create_position_events` to `_create_pose_events`.